src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs

```diff
@@ -24,15 +24,8 @@ protected override async Task OnInitAsync()
     {
         await base.OnInitAsync();
 
-        if (EmailConfirmed)
+        if (EmailConfirmed is false)
```

Comment:
don't we need to handle the `else` part?

Reply:
We have two pages in design :

1.  Email Confirmation page
2. Email Confirmation Failed page

The page known as the `EmailConfirmation`  is actually part of the sign-up page which notifies the user that an email has been sent.

The `EmailConfirmationFailed`  is for when the user clicks on the link sent in the email and returns to the program. 
There is a condition on this page that checks the validity of the email link.
If it was wrong, we will show the user the design implemented for the email confirmation failed page.
But when the link is correct, only a simple button is displayed to direct the user to sign-in (We have nothing for this in design pages)

However, I think saying an explanatory text is enough to apply to this page as failed page reciprocal term...
@msynk What do you think?

Reply:
are you sure about this information?
AFAIK there is no `EmailConfirmationFailed` page in the project. and the `EmailConfirmation` page is not part of the signup page and will show after clicking on the link provided in the email (it actually gets redirected by the server to this page).